### ISSUE TITLE
Content of the graph and tooltip is not clearly when zoom is set to 400%.

### ISSUE THREAD

**Issue by ashish315** [highcharts/highcharts#15441]
#### Expected behaviour
The content available at the tooltip and the graph should be visible properly even the web page is zoomed to 400%. The page should get scrolled as soon as the user navigates to the graph.

#### Actual behaviour
The content available at tooltip and graph is not visible properly when the system zoom is at 400%. The page didn't get scrolled when the user navigates at the graph.

#### Pre-Requisites:
1. Go to settings > Select 'System' settings > Display Resolution > Select 1280*768.
2. Go to browser settings > Zoom the page to  400% or 200%

#### Live demo with steps to reproduce

1. Open the URL: [https://www.highcharts.com/demo/line-basic](url)
2. Now, zoom your browser to 400%.
3. Now, navigate to the graph and observe the issue.

#### Note
Same issue observed when the zoom is at 200%.

#### Affected browser(s)
EDGE Anaheim

![image](https://user-images.githubusercontent.com/60913131/113671265-c1e1d200-96d3-11eb-9b8a-b299a77eb295.png)

**Comment by msftedad** [comment]
Hi Team, is there any update on this bug?

**Comment by oysteinmoseng** [comment]
Hi @msftedad, do I understand correctly that the issue is the page not scrolling up/down as you navigate within the chart?

**Comment by msftedad** [comment]
Hi @oysteinmoseng, there are 2 issues and both of them occur when the system zoom is at 400%. Hope this clarifies any doubts.
1. The content available at tooltip and graph is not visible properly when the system zoom is at 400%.
2. The page didn't get scrolled when the user navigates at the graph when the system zoom is at 400%.

**Comment by oysteinmoseng** [comment]
@msftedad Thanks, could you clarify what you mean by the first issue?
> The content available at tooltip and graph is not visible properly when the system zoom is at 400%

On my system this seems like a scrolling issue only, where the tooltip is not correctly in view - is this what is happening for you as well, or are there additional issues with the content?

**Comment by msftedad** [comment]
@oysteinmoseng 

The tooltip content is not visible and the graph is also not visible. Yes, its a scrolling issue but if a horizontal scroll is given to entire page, then it will be failed according to MAS 1.4.10 - reflow.
